packages/backend/src/services/UserService.ts

```diff
@@ -1041,6 +1050,28 @@ export class UserService {
         });
     }
 
+    async loginToOrganization(
+        userUuid: string,
+        loginMethod: string /* TODO make enum */,
+    ): Promise<
+        Pick<
+            LightdashUser,
+            'organizationUuid' | 'organizationCreatedAt' | 'organizationName'
+        >
+    > {
+        const organizations = await this.userModel.getOrganizationsForUser(
+            userUuid,
+        );
+        if (organizations.length === 0) {
+            throw new NotExistsError('User not part of any organization');
+        } else if (organizations.length > 1) {
+            throw new ForbiddenError('User is part of multiple organizations');
+        }
+        // TODO check valid login methods allowed in org
```

Comment:
afaik we don't have this

Reply:
Yep this is fine as a placeholder. 

There are valid cases where it's ok not to have an organisation. For example on first registering. So this error needs to be easy to catch or the function should return undefined